### PR TITLE
THEBIGBUGFIX: Fix form styles

### PR DESCRIPTION
**Pull request by taylortom**

### CODE REVIEW

File: frontend/src/core/scaffold/less/forms.less [hunk 1]
```diff
@@ -1,109 +1,106 @@
...
 }
 
-.field-object {
-	.legend {
-		cursor: pointer;
-	}
+.field-object .legend {
+    cursor: pointer;
+  }
 }
```
Comment (canstudios-nicolaw) [inline frontend/src/core/scaffold/less/forms.less]: Extra }

**Comment by canstudios-nicolaw** [comment]
Other than the small comment I made above it looks good to me :)

File: frontend/src/less/sharedStyles.less [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
-    > .inner {
-      padding: 30px;
+  max-width: 800px;
+  margin: 20px;
+  border: 1px solid #d6f2f9;
+  background-color: #fff;
+  > .inner {
+    padding: 30px;
+    form {
+      .form-group {
```
Comment (tomgreenfield) [inline frontend/src/less/sharedStyles.less]: We _could_ remove the unnecessary nesting here, i.e.
```less
form .form-group {
  // styles
}
```

**Comment by taylortom** [comment]
@tomgreenfield done

**Comment by taylortom** [comment]
@tomgreenfield have removed the uneven padding

**Pull request merged**